toolbox/issuer.go

```diff
@@ -60,13 +59,10 @@ func NewIssuerManager(config keycloak.Config, keyContextIssuerDomain interface{}
 	}, nil
 }
 
-func (im *issuerManager) GetIssuer(ctx context.Context) (OidcVerifierProvider, error) {
-	if rawValue := ctx.Value(im.keyContextIssuerDomain); rawValue != nil {
-		// The issuer domain has been found in the context
-		issuerDomain := getProtocolAndDomain(rawValue.(string))
-		if issuer, ok := im.domainToIssuer[issuerDomain]; ok {
-			return issuer, nil
-		}
+func (im *issuerManager) GetIssuer(issuer string) (OidcVerifierProvider, error) {
```

Comment:
Please find another name for the parameter: it does not make sens to call a method named GetIssuer(...) while providing a parameter named issuer. Or perhaps should you rename GetIssuer which is returning an OidcVerifierProvider ?